docs/pages/contributing.md

```diff
@@ -1,58 +1,96 @@
 ---
-title: Contributing
-description: This is a documentation page.
+title: Contributing to a Project
+description: Guidance for Contributing to CrowdStrike Open Source Projects
 permalink: /contributing/
 
 layout: post
 sidenav: contributing
 subnav:
   - text: Welcome
-    href: '#welcome-to-the-crowdstrike-community'
-  - text: Using Git
-    href: '#using-git'
+    href: '#welcome'
+  - text: How do I make a contribution?
+    href: '#how-do-i-make-a-contribution'
+  - text: Where can I go for help?
+    href: '#where-can-i-go-for-help'
+  - text: What does the Code of Conduct mean for me?
+    href: '#what-does-the-code-of-conduct-mean-for-me'
 ---
 
-## Welcome to the CrowdStrike Community
+# Welcome!
+CrowdStrike is still new to open source ourselves, and we recognize contributing to a project is no small feat! The guidance here aspires to help onboard new community members into how CrowdStrike-led projects tend to operate, and by extension, make the contribution process easier.
 
-Hello and welcome to the CrowdStike community on GitHub! Any and all contributions are welcome. The community may not always merge or act on something, but that is okay!
-Please keep engaging us.
+# How do I make a contribution?
+Never made an open source contribution before? Wondering how contributions work in CrowdStrike projects? Here's a quick rundown!
```

Comment:
```suggestion
Never made an open source contribution before? Wondering how contributions work in CrowdStrike projects? Here is a quick rundown!
```

While contractions are part of the english language, IMO it is better not to have them in web content for clarity.